src/test/kotlin/com/personio/synthetics/config/BrowserTestConfigTest.kt

```diff
@@ -45,6 +49,53 @@ internal class BrowserTestConfigTest {
         }
     }
 
+    @Test
+    fun `advancedScheduling function sets the advanced scheduling in the test config`() {
+        browserTest.advancedScheduling(
+            Timeframe(
+                from = LocalTime.of(0, 1),
+                to = LocalTime.of(23, 59),
+                DayOfWeek.MONDAY,
+                DayOfWeek.SUNDAY
+            ),
+            timezone = ZoneId.of("Europe/Dublin")
+        )
+
+        assertEquals(1, browserTest.options.scheduling.timeframes[0].day)
+        assertEquals("00:01", browserTest.options.scheduling.timeframes[0].from)
+        assertEquals("23:59", browserTest.options.scheduling.timeframes[0].to)
+        assertEquals(7, browserTest.options.scheduling.timeframes[1].day)
+        assertEquals("00:01", browserTest.options.scheduling.timeframes[1].from)
+        assertEquals("23:59", browserTest.options.scheduling.timeframes[1].to)
+        assertEquals("Europe/Dublin", browserTest.options.scheduling.timezone)
+    }
+
+    @Test
+    fun `advancedScheduling function sets the advanced scheduling with default timezone in the test config`() {
+        browserTest.advancedScheduling(
+            Timeframe(
+                from = LocalTime.of(0, 1),
+                to = LocalTime.of(23, 59),
+                DayOfWeek.MONDAY
+            )
+        )
+
+        assertEquals("Etc/UTC", browserTest.options.scheduling.timezone)
```

Comment:
This value depends on where the test actually runs in github and can become flaky. It would be better to do a not null and not empty check  for the value

Reply:
Fixed